DataModel/CctDbContext.cs

```diff
@@ -486,7 +486,7 @@ private void SeedData(ModelBuilder builder)
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             if (optionsBuilder.IsConfigured) return;
-            optionsBuilder.UseSqlServer(@"Server=(localdb)\mssqllocaldb;Database=CCT;Trusted_Connection=True;");
+            optionsBuilder.UseSqlServer(@"Server=LAPTOP-81CBV0VK\SQLEXPRESS;Database=CCTData;Trusted_Connection=True;");
```

Comment:
This needs to be changed back to 
optionsBuilder.UseSqlServer(@"Server=(localdb)\mssqllocaldb;Database=CCT;Trusted_Connection=True;");